pkg/instancegroups/rollingupdate.go

```diff
@@ -111,6 +111,13 @@ func (c *RollingUpdateCluster) RollingUpdate(groups map[string]*cloudinstances.C
 		wg.Wait()
 	}
 
+	// Do not continue update if bastion(s) failed
+	for _, err := range results {
+		if err != nil {
+			return err
+		}
```

Comment:
We should get an error here about the bastion- could you drop a verbose error log that explains that the rolling update is getting bounced because of the error in bastion? or too much?